main.tf

```diff
@@ -69,14 +69,14 @@ locals {
 [
   {
     "name": "${local.target_container_name}",
-    "image": "nginx:stable",
+    "image": "golang:1.11-stretch",
```

Comment:
How about using an Alpine image since it'll be much smaller? E.g., `1.11-alpine`

Reply:
Cool.  Switched to alpine and uses `/bin/sh`.